deploy/osd-delete-backplane-script-resources/10-delete-backplane-script-resources.rbac.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: osd-delete-backplane-script-resources-sa
+  namespace: openshift-backplane-managed-scripts
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: osd-delete-backplane-script-resources-role
+  namespace: openshift-backplane-managed-scripts
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - pods
+      - serviceaccounts
+    verbs:
+      - get
+      - list
+      - delete
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: osd-delete-backplane-script-resources-rb
+  namespace: openshift-backplane-managed-scripts
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: osd-delete-backplane-script-resources-role
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: User
```

Comment:
Shouldn't this be ServiceAccount here?

Reply:
Copied from other files and it worked... Seems we can specify it as a user then use a long name `system:serviceaccount:<ns>:<name>`.  But according to https://kubernetes.io/docs/reference/access-authn-authz/rbac/  we should specify it as `ServiceAccount` as the official documented way.   I will change that.